src/options.hpp

```diff
@@ -308,6 +308,9 @@ struct options_t
     //  Disconnect msg
     std::vector<unsigned char> disconnect_msg;
     bool can_recv_disconnect_msg;
+
+    //  This option removes several delays caused by scheduling, interrupts and context switching.
+    int busy_poll;
```

Comment:
Initialization in the class constructor for this class variable is missing

```
==17121== Conditional jump or move depends on uninitialised value(s)
==17121==    at 0x4EC2346: zmq::tcp_open_socket(char const*, zmq::options_t const&, bool, bool, zmq::tcp_address_t*) (tcp.cpp:417)
==17121==    by 0x4EC43D9: zmq::tcp_listener_t::create_socket(char const*) (tcp_listener.cpp:106)
==17121==    by 0x4EC45D3: zmq::tcp_listener_t::set_local_address(char const*) (tcp_listener.cpp:179)
==17121==    by 0x4EAE77F: zmq::socket_base_t::bind(char const*) (socket_base.cpp:653)
==17121==    by 0x4EDA4B6: zmq_bind (zmq.cpp:326)
==17121==    by 0x4019DB: test_localhost() (test_system.cpp:72)
==17121==    by 0x406138: UnityDefaultTestRun (unity.c:1339)
==17121==    by 0x401BC2: main (test_system.cpp:114)
```

https://github.com/zeromq/libzmq/blob/5c22bfd8c203acf128d5d8348710fa8351b0221c/src/options.cpp#L257

Reply:
this is still missing, and breaks the valgrind build